### ISSUE TITLE
Some dialogs (add video, update) no longer work after update; app hang

### ISSUE THREAD

**Issue by ceonelson** [axcore/tartube#307]
I updated from 2.3.110 to 2.3.203 by extracting the zip as previously done many times and now I can no longer add videos and some other errors:

Add video button errors:
Traceback (most recent call last):
```
  File "C:\Users\Owner\Documents\___Apps\Tartube\msys64\home\user\tartube\tartube\mainapp.py", line 19806, in on_menu_add_video
    dialogue_win = mainwin.AddVideoDialogue(self.main_win_obj)
  File "C:\Users\Owner\Documents\___Apps\Tartube\msys64\home\user\tartube\tartube\mainwin.py", line 25296, in __init__
    and media_data_obj.restrict_mode == 'open' \
AttributeError: 'Folder' object has no attribute 'restrict_mode'
```

Add channel button error:
```
Traceback (most recent call last):
  File "C:\Users\Owner\Documents\___Apps\Tartube\msys64\home\user\tartube\tartube\mainapp.py", line 19401, in on_menu_add_channel
    and container_obj.restrict_mode == 'open':
AttributeError: 'Folder' object has no attribute 'restrict_mode'
```

Add playlist button error:
```
Traceback (most recent call last):
  File "C:\Users\Owner\Documents\___Apps\Tartube\msys64\home\user\tartube\tartube\mainapp.py", line 19664, in on_menu_add_playlist
    and container_obj.restrict_mode == 'open':
AttributeError: 'Folder' object has no attribute 'restrict_mode'
```

Tried to update yt-dlp:
```
Traceback (most recent call last):
  File "C:\Users\Owner\Documents\___Apps\Tartube\msys64\home\user\tartube\tartube\mainapp.py", line 20661, in on_menu_update_ytdl
    self.update_manager_start('ytdl')
  File "C:\Users\Owner\Documents\___Apps\Tartube\msys64\home\user\tartube\tartube\mainapp.py", line 9405, in update_manager_start
    self.main_win_obj.sensitise_check_dl_buttons(False, update_type)
  File "C:\Users\Owner\Documents\___Apps\Tartube\msys64\home\user\tartube\tartube\mainwin.py", line 4101, in sensitise_check_dl_buttons
    self.custom_dl_media_button.set_label('...')
AttributeError: 'NoneType' object has no attribute 'set_label'
```

As I saw a folder object in the add video error I tried to go to a folder that only had 1 item in it and add a video and it caused the dreaded (not responding) so I couldn't get any error in the command prompt.

I had verified the DB and no errors were found.

Let me know if you want me to try anything out!

Thanks again for making such an awesome app Axcore!

**Comment by mbnoimi** [comment]
The same bug occurs in Linux. tartube no longer able to download anything!

```
$ python3 tartube

(tartube:152624): Gdk-CRITICAL **: 06:01:02.075: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed
Traceback (most recent call last):
  File "/home/hpc-laptop/.tartube/tartube/mainapp.py", line 17510, in script_slow_timer_callback
    self.download_manager_start(
  File "/home/hpc-laptop/.tartube/tartube/mainapp.py", line 8839, in download_manager_start
    self.download_manager_continue(
  File "/home/hpc-laptop/.tartube/tartube/mainapp.py", line 8935, in download_manager_continue
    self.update_manager_start('ytdl')
  File "/home/hpc-laptop/.tartube/tartube/mainapp.py", line 9405, in update_manager_start
    self.main_win_obj.sensitise_check_dl_buttons(False, update_type)
  File "/home/hpc-laptop/.tartube/tartube/mainwin.py", line 4101, in sensitise_check_dl_buttons
    self.custom_dl_media_button.set_label('...')
AttributeError: 'NoneType' object has no attribute 'set_label'
Traceback (most recent call last):
  File "/home/hpc-laptop/.tartube/tartube/mainapp.py", line 20084, in on_menu_download_all
    self.download_manager_start('real')
  File "/home/hpc-laptop/.tartube/tartube/mainapp.py", line 8839, in download_manager_start
    self.download_manager_continue(
  File "/home/hpc-laptop/.tartube/tartube/mainapp.py", line 8935, in download_manager_continue
    self.update_manager_start('ytdl')
  File "/home/hpc-laptop/.tartube/tartube/mainapp.py", line 9405, in update_manager_start
    self.main_win_obj.sensitise_check_dl_buttons(False, update_type)
  File "/home/hpc-laptop/.tartube/tartube/mainwin.py", line 4101, in sensitise_check_dl_buttons
    self.custom_dl_media_button.set_label('...')
AttributeError: 'NoneType' object has no attribute 'set_label'
hpc-laptop@hpc-laptop:~/.tartube/tartube$ 
```

**Comment by axcore** [comment]
All fixed in v2.3.206. Sorry for the inconvenience.
